src/OpenLoco/Entities/EntityManager.cpp

```diff
@@ -13,13 +15,68 @@ namespace OpenLoco::EntityManager
     loco_global<uint16_t[numEntityLists], 0x00525E4C> _listCounts;
     loco_global<Entity[maxEntities], 0x006DB6DC> _entities;
     loco_global<EntityId_t[0x40001], 0x01025A8C> _entitySpatialIndex;
+    loco_global<uint32_t, 0x01025A88> _entitySpatialCount;
     static loco_global<string_id, 0x009C68E6> gGameCommandErrorText;
     constexpr size_t _entitySpatialIndexNull = 0x40000;
 
     // 0x0046FDFD
     void reset()
```

Comment:
Could you add a few comments to this function to give an outline as to what steps are going on?